File: 3st/Jaeyoung/n의 배수 고르기.js
function solution(n, numlist) {
    var answer = [];
    var j = 0;
    for ( var i = 0 ; i < numlist.length ; i++ ) 
        if ( numlist[i]%n == 0 ) answer[j++] = numlist[i];
    return answer;
}

File: 4st/Jisoo/programmer/n의 배수 고르기.java
class Solution {
    public int[] solution(int n, int[] numlist) {
        int a=0;
        for(int i=0;i<numlist.length;i++){
            if(numlist[i]%n==0){
                a++;
            }
        }
        int [] answer=new int[a];
        int k=0;
        for(int i=0;i<numlist.length;i++){
            if(numlist[i]%n==0){
                answer[k]=numlist[i];
                k++;
            }
        }
        return answer;
    }
}